beginner_source/flava_finetuning_tutorial.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+"""
+TorchMultimodal Tutorial: Finetuning FLAVA
+============================================
+"""
+
+######################################################################
+# Multimodal AI has recently become very popular owing to its ubiquitous
+# nature, from use cases like image captioning and visual search to more
+# recent applications like image generation from text. **TorchMultimodal
+# is a library powered by Pytorch consisting of building blocks and end to
+# end examples, aiming to enable and accelerate research in
+# multimodality**.
+# 
+# In this tutorial, we will demonstrate how to use a **pretrained SoTA
+# model called** `FLAVA <https://arxiv.org/pdf/2112.04482.pdf>`__ **from
+# TorchMultimodal library to finetune on a multimodal task i.e. visual
+# question answering** (VQA).
+# 
+
+
+######################################################################
+# Installations
+#
+# We will use TextVQA dataset and bert tokenizer from HuggingFace for this
+# tutorial. So you need to install datasets and transformers in addition to TorchMultimodal.
+# When running this tutorial in Google Colab, install the required packages
+# by uncommenting the following:
+#
+"""
+!pip install torchmultimodal-nightly
+!pip install datasets
+!pip install transformers
+"""
+
+######################################################################
+# For this tutorial, we treat VQA as a classification task. So we need to
+# download the vocab file with answer classes and create the answer to
+# label mapping.
+#
+# We also load the `textvqa
+# dataset <https://arxiv.org/pdf/1904.08920.pdf>`__ from HuggingFace
+#
+
+with open("data/vocabs/answers_textvqa_more_than_1.txt") as f:
```

Comment:
Not sure where this file is from. Does the user need to download this or can it be generated from the dataset?

Reply:
It's downloaded, see `Makefile` changes

Reply:
The Makefile download  works for building the tutorial, but not for a user running this tutorial on a colab/local notebook? Should the `tutorial.py` have a comment with the vocab url?